app/src/common/shared/com/igalia/wolvic/VRBrowserActivity.java

```diff
@@ -284,6 +285,11 @@ protected void onCreate(Bundle savedInstanceState) {
 
         SettingsStore.getInstance(getBaseContext()).setPid(Process.myPid());
         ((VRBrowserApplication)getApplication()).onActivityCreate(this);
+
+        if (!DeviceType.isHVRBuild() && SettingsStore.getInstance(getBaseContext()).isTelemetryEnabled()) {
```

Comment:
Is HVR the only platform we won't add telemetry support for ? I guess it's because it's kind of legacy. In any case, I believe a comment to clarify the reasons would make sense.

Reply:
HVR uses it's only telemetry with its own Huawei library. We should be able to add our own telemetry but that would require additional changes because right now the code asumes a single telemetry provider.